### PR TITLE
Logo

### PR DESCRIPTION
**Pull request by Sleptco**
- [DEMO LINK](https://Sleptco.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Sleptco.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
As I understand, you implemented the solution in the other PR. Write in the chat if I am wrong

**Comment by Sleptco** [comment]
> As I understand, you implemented the solution in the other PR. Write in the chat if I am wrong
Вітаю. Не розумію питання з приводу іншого пул реквесту.

**Comment by Sleptco** [comment]
Нагадайте як вірно зробити  DEMO LINK
 TEST REPORT LINK десь було відео неможу знайти. Дякую.

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://Sleptco.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://Sleptco.github.io/layout_moyo-header/report/html_report/)
```
Comment (Viktor-Kost) [inline readme.md:4]: Add this links to PR description

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,64 @@
...
+    <header class="header">
+      <div class="wrapper">
+        <a href="#home"
+          class="logo_home">
+          <img
+          class="logo"
+          src="images/logo.png"
+          alt="Moyo"
+          >
+        </a>
```
Comment (Viktor-Kost) [inline src/index.html:24]: apply code styling recommendations all over the project
```suggestion
        <a 
          href="#home"
          class="logo_home"
        >
          <img
            class="logo"
            src="images/logo.png"
            alt="Moyo"
          >
         </a>
```

File: src/index.html [hunk 3]
```diff
@@ -7,8 +7,64 @@
...
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
```
Comment (Viktor-Kost) [inline src/index.html:16]: it's redundant wrapper
```suggestion
```

File: src/index.html [hunk 4]
```diff
@@ -7,8 +7,64 @@
...
+        <a href="#home"
+          class="logo_home">
+          <img
+          class="logo"
+          src="images/logo.png"
+          alt="Moyo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav_list">
```
Comment (Viktor-Kost) [inline src/index.html:26]: according to BEM methodology. Apply all over the project
```suggestion
        <nav class="nav">
          <ul class="nav__list">
```

File: src/index.html (hunk 2 above)
Comment (Viktor-Kost) [inline src/index.html:24]: is this link an element of header? Use BEM
```suggestion
        <a href="#home"
          class="header__logo-link">
          <img
          class="header__logo-img"
          src="images/logo.png"
          alt="Moyo"
          >
        </a>
```

File: src/style.css [hunk 5]
```diff
@@ -1 +1,65 @@
...
+
+.logo_home {
+  position: relative;
+  left: 50px;
+  top: 10px;
+}
+
+.nav_list {
+  display: flex;
+  /* flex-direction: row; */
```
Comment (Viktor-Kost) [inline src/style.css:27]: remove comments
```suggestion
```

File: src/style.css [hunk 6]
```diff
@@ -1 +1,65 @@
...
+  font-family: Roboto, sans-serif;
```
Comment (Viktor-Kost) [inline src/style.css:28]: apply font-family to html tag

File: src/style.css [hunk 7]
```diff
@@ -1 +1,65 @@
...
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+
+}
+
+.nav_item {
+  margin-right: 20px;
```
Comment (Viktor-Kost) [inline src/style.css:37]: last item shouldn't have a margin

File: src/style.css [hunk 8]
```diff
@@ -1 +1,65 @@
...
+  height: 100%;
+}
+
+.logo_home {
+  position: relative;
+  left: 50px;
+  top: 10px;
+}
+
+.nav_list {
```
Comment (Viktor-Kost) [inline src/style.css:25]: reset default margins for list

File: src/style.css [hunk 9]
```diff
@@ -1 +1,65 @@
...
+}
+
+.nav_link {
+  text-decoration: none;
+  cursor: pointer;
+  position: relative;
+}
+
+.nav_link:hover {
+  color: #00acdc;
```
Comment (Viktor-Kost) [inline src/style.css:48]: create variables for repeated colors

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Hi!)
Check a design, because now it looks incorrect and try to pass tests

Screenshot 2023-02-22 at 15 35 33
Screenshot 2023-02-22 at 15 35 13

File: src/index.html [hunk 10]
```diff
@@ -7,8 +7,65 @@
...
+          href="#home"
+          class="logo_home">
+          <img
+          class="logo"
+          src="images/logo.png"
+          alt="Moyo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav_list">
```
Comment (SerhiiKirik) [inline src/index.html:27]: we mark the Element in the BEM like "block__element"
```suggestion
          <ul class="nav__list">
```

File: src/index.html [hunk 11]
```diff
@@ -7,8 +7,65 @@
...
+            <li class="nav_item">
+              <a href="#smartphones" class="nav_link">Smartphones</a>
+            </li>
+
+            <li class="nav_item">
+              <a
+               href="#laptops & computers"
+               class="nav_link"
+               data-qa="hover"
+               >Laptops & Computers
```
Comment (SerhiiKirik) [inline src/index.html:46]:
```suggestion
              <a
               href="#laptops & computers"
               class="nav_link"
               data-qa="hover"
             >
               Laptops & Computers
```

File: src/style.css (hunk 7 above)
Comment (SerhiiKirik) [reply src/style.css:32]: use nth-child or last-child here

File: src/style.css [hunk 12]
```diff
@@ -1 +1,71 @@
...
+  margin-right: 20px;
+}
+
+.nav_item:last-child {
+  margin-right: 0;
+}
+
+.nav_link {
+  position: relative;
+  /* padding: 22.5px 0; */
```
Comment (SerhiiKirik) [inline src/style.css:41]:
```suggestion
```

File: src/style.css [hunk 13]
```diff
@@ -1 +1,71 @@
...
+  line-height: 14px;
+  display: flex;
```
Comment (SerhiiKirik) [inline src/style.css:43]: organize your CSS styles 
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)

File: src/style.css [hunk 14]
```diff
@@ -1 +1,71 @@
...
+  height: 60px;
+  align-items: center;
+  width: max-content;
+  font-size: 12px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
```
Comment (SerhiiKirik) [inline src/style.css:49]: move to html tag
```suggestion
```

File: src/style.css [hunk 15]
```diff
@@ -1 +1,71 @@
...
+.logo_home {
+  width: 40px;
+  height: 40px;
+}
+
+.nav_list {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
```
Comment (SerhiiKirik) [inline src/style.css:28]: reset this like
ul, li {
}
```suggestion
```

File: src/index.html [hunk 16]
```diff
@@ -7,8 +7,65 @@
...
+        <a
+          href="#home"
+          class="logo_home">
+          <img
+          class="logo"
+          src="images/logo.png"
+          alt="Moyo"
+          >
+        </a>
+        <nav class="nav">
```
Comment (SerhiiKirik) [inline src/index.html:26]: Loot at 
https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md
and
image

```suggestion
        <a
          href="#home"
          class="logo_home"
        >
          <img
            class="logo"
            src="images/logo.png"
            alt="Moyo"
          >
        </a>
        
        <nav class="nav">
```

File: src/index.html (hunk 3 above)
Comment (SerhiiKirik) [reply src/index.html:16]: Please, don't ignore comments / If you have a question - write here or in the slack chat

**Review by SerhiiKirik — CHANGES_REQUESTED** [review]
Good job 🔥 
Let's fix some comments, improve code & mode on 🧑‍💻

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Let's connect in your personal chat, it will be simpler to figure out there) Please invite me

File: .linthtmlrc.json [hunk 17]
```diff
@@ -1,3 +1,49 @@
...
+  ],
+  "tag-name-lowercase": true,
+  "tag-name-match": true,
+  "tag-self-close": "never",
+  "tag-close": true,
+  "text-ignore-regex": "&",
+  "title-no-dup": true,
+  "line-end-style": "lf",
+  "attr-new-line": 2,
+  "attr-name-style": "dash",
```
Comment (SerhiiKirik) [inline .linthtmlrc.json:47]: In the future - Do not add extra files to PR
To add the file to a commit use git add src/.

File: src/index.html [hunk 18]
```diff
@@ -7,8 +7,65 @@
...
+          </li>
+
+          <li class="nav_item">
+            <a
+               href="#laptops & computers"
+               class="nav_link"
+               data-qa="hover"
+               >
+               Laptops & Computers
+            </a>
```
Comment (SerhiiKirik) [inline src/index.html:48]:
```suggestion
      <a
        href="#home"
        class="header__logo-link"
      >
        <img
          class="header__logo-img"
          src="images/logo.png"
          alt="Moyo"
        >
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav_item">
            <a href="#apple" class="nav_link is-active">Apple</a>
          </li>

          <li class="nav_item">
            <a href="#samsung" class="nav_link">Samsung</a>
          </li>

          <li class="nav_item">
            <a href="#smartphones" class="nav_link">Smartphones</a>
          </li>

          <li class="nav_item">
            <a
               href="#laptops & computers"
               class="nav_link"
               data-qa="hover"
             >
               Laptops & Computers
            </a>
```

**Review by SerhiiKirik — APPROVED** [review]
Great 🚀

File: src/index.html (hunk 18 above)
Comment (SerhiiKirik) [reply src/index.html:48]: https://github.com/mate-academy/layout_html-form/blob/master/checklist.md

**Pull request closed without merging**